README.md

```diff
@@ -1,61 +1,87 @@
 # U6143_ssd1306
-## Preparation
+
+##  I2C
+Begin by enabling the I2C interface:
+
 ```bash
 sudo raspi-config
```

Comment:
Maybe worth mentioning that we can enable it without the ncurses interface?

via: `raspi-config nonint do_i2c 0`

Reply:
Same comment as above, it's another way, but it's not really within the scope of this to show all the ways to perform the administrative tasks... Also, I don't think the `nonint `commands are nominally documented - they're only known through a reading of the source code. But it's definitely an option for advanced administrators to go about the task.